### PR TITLE
Add Bibliopolis and Scottish Book Trade Index

### LINKED ISSUE #2: Maintainable command handling
Replace the current `if`/`else` trees with something more maintainable, for example using [cmd](https://docs.python.org/3/library/cmd.html), as requested in the comments on #1.

### LINKED ISSUE #3: Convert tests to pytest
As requested in review to #1.

### PR DESCRIPTION
**Pull request by tijmenbaarda**
* Add Bibliopolis (SRU API)
* Add Scottish Book Trade Index (custom JSON API)
* Use cmd2 for commandline loop (closes #2)
* Move from unittest to pytest (closes #3)

### CODE REVIEW

File: tests/test_srureader.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 TESTDATA = json.load(open(Path(__file__).parent / 'TESTDATA', 'r'))
 
 
-class SRUReaderTestCase(unittest.TestCase):
+class TestSRUReader:
     @patch('edpop_explorer.srureader.sruthi')
```
Comment (lukavdplas) [inline tests/test_srureader.py:13]: Since you are converting to pytest, is there a reason for keeping in this unit test function?

**Review by lukavdplas — APPROVED** [review]
Nice, the cmd2 implementation certainly looks clearer!

File: tests/test_srureader.py (hunk 1 above)
Comment (tijmenbaarda) [reply tests/test_srureader.py:13]: I thought pytest is used for unit tests as well? Or do you suggest a different set-up of the tests?

Comment (lukavdplas) [reply tests/test_srureader.py:13]: Oh, this was unclear. I meant that `patch` comes from the `unittest` library.

**Pull request merged**